Resources/Locale/en-US/_Impstation/xenoarchaeology/artifact-analyzer.ftl

```diff
@@ -0,0 +1,11 @@
+﻿analysis-console-info-natural-locked-value = [font="Monospace" size=11][color={ $state ->
+    [0] red]Unactivated
+    [1] lime]Activated
+    [2] plum]Current Activated
+    *[3] palevioletred]Current Unactivated
```

Comment:
I'm not a fan of how this reads? It feels awkward to me. Maybe split it into a "Current node" line and "activated" line?